matching-notification-api/src/main/java/org/phenotips/matchingnotification/storage/internal/DefaultMatchStorageManager.java

```diff
@@ -119,6 +121,8 @@
     private static final String HQL_QUERY_REMOTE_MATCHES =
         "select count(*) from DefaultPatientMatch as m where m.referenceServerId != '' or m.matchedServerId !=''";
 
+    private static final int ONE_HOUR_IN_MILLISECONDS = 1000 * 60 * 60 * 24;
```

Comment:
This is one _day_ in milliseconds.

Reply:
yes. sorry , that is what we need and what I meant to say